src/credential/sd_jwt_vc.rs

```diff
@@ -0,0 +1,94 @@
+use ssi::claims::{
+    jwt::{AnyClaims, JWTClaims},
+    sd_jwt::{RevealedSdJwt, SdJwtBuf},
+};
+
+#[uniffi::export]
+pub fn decode_reveal_sd_jwt(input: String) -> Result<String, SdJwtVcError> {
+    let jwt: SdJwtBuf = SdJwtBuf::new(input).map_err(|_| SdJwtVcError::InvalidSdJwt)?;
+    let revealed_jwt: RevealedSdJwt<AnyClaims> = jwt
+        .decode_reveal_any()
+        .map_err(|_| SdJwtVcError::JwtDecoding)?;
+    let claims: &JWTClaims = revealed_jwt.claims();
+    serde_json::to_string(claims).map_err(|_| SdJwtVcError::Serialization)
```

Comment:
This `.map_err` could be an `unwrap`. The `JWTClaims` is compatible with the JSON data model so there is not reason the serialization should fail. But that's a detail, not a blocker.